include/h2o.h

```diff
@@ -275,6 +275,28 @@ typedef struct st_h2o_protocol_callbacks_t {
     int (*foreach_request)(h2o_context_t *ctx, int (*cb)(h2o_req_t *req, void *cbdata), void *cbdata);
 } h2o_protocol_callbacks_t;
 
+enum h2o_status_handler_type {
+    H2O_STATUS_HANDLER_SIMPLE,
+    H2O_STATUS_HANDLER_PER_THREAD,
+};
+typedef h2o_iovec_t (*simple_status_handler_cb)(h2o_globalconf_t *gconf, h2o_mem_pool_t *pool);
+typedef struct h2o_status_handler {
```

Comment:
Please rename the struct to `struct st_h2o_status_handler_t` (since others have `st_` prefix and `_t` suffix).